_posts/2017-05-17-olx.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OLX
+sourceURL: https://developers.google.com/web/showcase/2017/olx
+image: /images/olx.png
+tags:
+- App Shell
+- Home Screen
+- IndexedDB
+- Service Worker
+- Notifications
+- Media
+- E-commerce
+
+---
+
+OLX experienced **250% more re-engagement** using push notifications and **146% higher click-through rate on ads**. The PWA takes **23% less time** to interactive resulting in **80% lower** bounce rates.
```

Comment:
Perhaps this is my case against bolding more than just company/app name & figures only? Feels like we might be on a slippery slope where pretty soon _everything get bolded_. 😅